deploy/docker-compose.graph.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.1'
+
+volumes:
+  postgres:
+    driver: local
+
+services:
```

Comment:
We need to change `STAKEWISE_SUBGRAPH_URL`, `UNISWAP_V3_SUBGRAPH_URL`, `ETHEREUM_SUBGRAPH_URL` URLs in case graph is deployed locally

Reply:
The same as above, we can't changed it based on conditions, perhaps it's better to just add a documentation for service launch and configuration.